src/NoteDescription.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function NoteDescription({
+  notice,
+  index,
+  completeNotice,
+  removeNotice,
+}) {
+  return (
+    <>
+      <MyNotes
+        style={{
+          textDecoration: notice.isCompleted ? 'line-through' : '',
+          textDecorationThickness: notice.isCompleted ? '0.1em' : '',
```

Comment:
Vll lieber in die styled-components?

Reply:
Ja, ist ein sehr guter Hinweis Hajo. Danke dir.